### PR TITLE
Small components adjustments for future-header

### PR DESCRIPTION
**Pull request by lbebber**
#### What is the purpose of this pull request?
Adjusts defaults of components for better use on the new header.

#### Types of changes
- [ ] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,12 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added 
+- Allow adding link to logo.
+
+### Changed
+- Set w-100 to search bar and add padding
```
Comment (klzns) [inline CHANGELOG.md]:
```suggestion
- Set w-100 to search bar and add padding.
```

TOC

File: react/components/Logo/index.js [hunk 2]
```diff
@@ -55,58 +69,64 @@ class Logo extends Component {
...
+              />
+              <path d="M0 0h24v24H0z" fill="none" />
+            </svg>
+          </div>
+        </div>
+      )
+    }
+
+    return (
+      <div className="pv4 ph6">
```
Comment (klzns) [inline react/components/Logo/index.js]: Can we place this paddings in the div bellow?

File: react/components/SearchBar/components/SearchBar.js [hunk 3]
```diff
@@ -38,10 +39,18 @@ export default class SearchBar extends Component {
       />
     )
 
-    const mainClasses = classNames(styles.searchBarContainer)
+    const mainClasses = `w-100 mw7 pv4 ${classNames(styles.searchBarContainer)}`
```
Comment (klzns) [inline react/components/SearchBar/components/SearchBar.js]:
```suggestion
    const mainClasses = classNames('w-100 mw7 pv4', styles.searchBarContainer)
```

This and I would also just inline it in the div, like: `<div className={classNames(..`

File: react/components/Logo/index.js [hunk 4]
```diff
@@ -55,58 +69,64 @@ class Logo extends Component {
...
         <div className={`${logoClassNames} store-logo`}>
           <svg
+            width={width}
```
Comment (klzns) [inline react/components/Logo/index.js]: I would create a separate file for those svgs. Then just would them like:

```jsx
<PlaceholderLogo />
```

Or something like that

File: CHANGELOG.md (hunk 1 above)
Comment (jgfidelis) [reply CHANGELOG.md]: :developers

File: react/components/Logo/Placeholder.js [hunk 5]
```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+
+export default ({ width, height }) => (
```
Comment (matheusps) [inline react/components/Logo/Placeholder.js]: Would not be better if these svgs were on `store-icons`? 🤔

Comment (lbebber) [reply react/components/Logo/Placeholder.js]: @matheusps This PR was just meant to add a padding around the Logo. All these store-components need to be reviewed, and these kind of changes will be done in future PRs

**Pull request merged**